### ISSUE TITLE
Bug: Text should be aligned towards left in card template list

### ISSUE THREAD

**Issue by harshilsharma63** [mattermost/focalboard#3849]
<img width="340" alt="Screenshot 2022-09-15 at 2 09 10 PM" src="https://user-images.githubusercontent.com/18575143/190357286-0b239d29-d69c-46dc-a818-4a84bc2c3bb8.png">

**Comment by sbishel** [comment]
@asaadmahmood - hopefully this is quick based on your other recent UX fixes. Not sure if we should be assigning to you still or not?

**Comment by asaadmahmood** [comment]
@sbishel Yup, feel free to assign things to me, I'll try to get any UX UI fixes in even over the weekend so we're good for the release.
